i8n/srt-to-mediatree-format.py

```diff
@@ -65,19 +65,24 @@ def parse_csv_without_headers(file_path, encoding='utf-8'):
     
     return data
 
-def convert_datetime_to_timestamp(date_str):
+def convert_datetime_to_timestamp(date_str, date_format="%m/%d/%Y %H:%M:%S"):
     """Convert date string to UNIX timestamp"""
+    formats_T2_2025 = ["%d/%m/%Y %H:%M:%S", "%d/%m/%Y %H:%M"]
+    if date_format in formats_T2_2025:
+        if len(date_str) == 19:
```

Comment:
quick and dirty to be able to handle date with seconds and without seconds inside the same CSV file